### PR TITLE
Complete the homework №3

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

File: src/main/java/hw_3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.Arrays;
+public class Task1 {
+public static void main(String[] args) {
+    //task1
+    double[] array = {3.43, 12.1212, 12.22, 2.29};
+    System.out.println(Arrays.toString(array));
+    for(double example : array ){
+    String result = example + ";";
+    System.out.println(result);
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: Your output should be in one line - "3.43;12.1212;12.22;2.29"
For now you output is 
Screenshot_9

File: src/main/java/hw_3/Task1.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    for (String result : word ){
+    System.out.println(result);
+    }
+
+    //task3
+    String task = "Я хочу перенесення рядків";
+    System.out.println(task);
+    System.out.println(task.replace(" ","\n"));
+}
+}
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java:31]: The tabulation is incorrect.
THe correct view is:
Screenshot_10

I'd suggest to set up a Ctrl + Alt + L key combination

File: src/main/java/hw_3/Task1.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    //task2
+    String sentence = "Мама мила раму";
+    System.out.println(sentence);
+    String[] word = sentence.split( " ");
+    for (String result : word ){
+    System.out.println(result);
+    }
+
+    //task3
+    String task = "Я хочу перенесення рядків";
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: 'task' is a bad name for this variable

File: src/main/java/hw_3/Task2.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        values[2] = (int) c;
+        System.out.println(Arrays.toString(values));
+        //task4
+        int[][] arrayTwins = new int[3][2];
+        System.out.println(Arrays.deepToString( arrayTwins));
+        arrayTwins[0][1] = 17;
+        arrayTwins[2][0] = 5;
+        arrayTwins[1][1] = 24;
+        arrayTwins[0][0] = 14;
+        System.out.println(Arrays.deepToString(arrayTwins));
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java:41]: Your array isn't full filed. Please fill out this array completely
Screenshot_11

Comment (natak8200) [reply src/main/java/hw_3/Task2.java:41]: fixed

File: src/main/java/hw_3/Task1.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_3/Task1.java:31]: fixed

File: src/main/java/hw_3/Task1.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 1 above)
Comment (natak8200) [reply src/main/java/hw_3/Task1.java]: fixed

Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: The implementation is incorrect. You need to create a separate empty string before cycle and make concatenation all arrays elements with created string inside cycle. Then after cycle is completed display the string result in console

File: src/main/java/hw_3/Task1.java (hunk 3 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: The new name is bad as well. When I open your code and read the variable name "rescheduled" I won't be able to understand what is the variable.
lets "rename" it as text or something like that

Comment (natak8200) [reply src/main/java/hw_3/Task1.java]: fixed

**Pull request merged**